test/parallel/begin/deitz/test_control_flow.good

```diff
@@ -1,31 +1,36 @@
-test_control_flow.chpl:81: error: continue to a named loop outside of a forall is not allowed from inside the forall
+test_control_flow.chpl:2: In function 'foo':
+test_control_flow.chpl:5: error: 'return' is not allowed in a 'begin' statement
+test_control_flow.chpl:6: error: 'break' is not allowed in a 'begin' statement
+test_control_flow.chpl:7: error: 'continue' is not allowed in a 'begin' statement
+test_control_flow.chpl:10: error: 'return' is not allowed in a 'cobegin' statement
+test_control_flow.chpl:11: error: 'break' is not allowed in a 'cobegin' statement
+test_control_flow.chpl:12: error: 'continue' is not allowed in a 'cobegin' statement
+test_control_flow.chpl:15: error: 'return' is not allowed in an 'on' statement
+test_control_flow.chpl:16: error: 'break' is not allowed in an 'on' statement
+test_control_flow.chpl:17: error: 'continue' is not allowed in an 'on' statement
+test_control_flow.chpl:21: error: 'return' is not allowed in a 'coforall' statement
+test_control_flow.chpl:22: error: 'break' is not allowed in a 'coforall' statement
+test_control_flow.chpl:23: error: 'continue' is not allowed in a 'coforall' statement
+test_control_flow.chpl:26: error: 'return' is not allowed in a 'forall' statement
+test_control_flow.chpl:27: error: 'break' is not allowed in a 'forall' statement
+test_control_flow.chpl:34: In iterator 'bar':
+test_control_flow.chpl:37: error: 'yield' is not allowed in a 'begin' statement
+test_control_flow.chpl:38: error: 'return' is not allowed in a 'begin' statement
+test_control_flow.chpl:39: error: 'break' is not allowed in a 'begin' statement
+test_control_flow.chpl:40: error: 'continue' is not allowed in a 'begin' statement
+test_control_flow.chpl:44: error: 'return' is not allowed in a 'cobegin' statement
+test_control_flow.chpl:45: error: 'break' is not allowed in a 'cobegin' statement
+test_control_flow.chpl:46: error: 'continue' is not allowed in a 'cobegin' statement
+test_control_flow.chpl:50: error: 'return' is not allowed in an 'on' statement
+test_control_flow.chpl:51: error: 'break' is not allowed in an 'on' statement
+test_control_flow.chpl:52: error: 'continue' is not allowed in an 'on' statement
+test_control_flow.chpl:57: error: 'return' is not allowed in a 'coforall' statement
+test_control_flow.chpl:58: error: 'break' is not allowed in a 'coforall' statement
+test_control_flow.chpl:59: error: 'continue' is not allowed in a 'coforall' statement
+test_control_flow.chpl:63: error: 'return' is not allowed in a 'forall' statement
+test_control_flow.chpl:64: error: 'break' is not allowed in a 'forall' statement
+test_control_flow.chpl:69: In function 'baz':
+test_control_flow.chpl:75: error: could not find label 'namedOuter' for 'continue' statement
+test_control_flow.chpl:72: note: stopped searching here because 'continue' statements are not allowed to jump outside a 'forall' statement
```

Comment:
It feels a bit unnecessary to say that we couldn't find the label, only to immediately follow that up with the notion that we couldn't use the label anyways even if we found it. Would it be possible to just say that 'continue' statements are not allowed in this situation?